end-to-end-tests/code-build/README.md

```diff
@@ -0,0 +1,33 @@
+# Setup CodeBuild for End to End Test
+
+We use CodeBuild to run quick end-to-end test. Below are steps to set it up:
+
+### Running `deploy` to setup blox application
+To avoid too broad permissions required for CodeBuild project, we setup blox project firstly.
+
+Make sure you have one AWS credential from the AWS account and used in **default** profile:
+
+```
+$ cat end-to-end-tests/code-build/end2end.properties >> ~/.gradle/gradle.properties
+./gradlew createBucket
+./gradlew deploy
+```
+
+### Create CodeBuild Project
+Before we create the project, you must connect the AWS account to the GitHub account. 
+To do this, use the AWS CodeBuild console to create a build project. When you use the console to connect (or reconnect) with GitHub, on the GitHub Authorize application page, for Organization access, choose Request access next to each repository you want AWS CodeBuild to be able to access. Choose Authorize application. 
```

Comment:
This is kind of hard to parse; maybe split out into a bullet list?

Reply:
yep, will update.